sql/core/src/main/scala/org/apache/spark/sql/SQLConf.scala

```diff
@@ -399,7 +399,7 @@ private[spark] object SQLConf {
     isPublic = false)
 
   val USE_SQL_AGGREGATE2 = booleanConf("spark.sql.useAggregate2",
-    defaultValue = Some(true), doc = "<TODO>")
+    defaultValue = Some(false), doc = "<TODO>")
```

Comment:
I turned off new aggregate Path here intentionally to check the revert didn't crash others which I didn't find out. Would discard this when the test success